app/controllers/api/news_controller.rb

```diff
@@ -60,6 +60,16 @@ def index
     respond_success serialized(@news_items)
   end
 
+
+  def show
+    # Ensure fresh response on every request
+    headers['Last-Modified'] = Time.now.httpdate
+
+    @news_item = NewsItem.published.find(params[:id])
+
+    respond_success serialized(@news_item)
+  end
```

Comment:
It's impossible to take some news from fetched news array because it does not contain all the news. So, It's needed to create this action to get particular news by id

Reply:
Perfect, this is correct, thanks 👍